deploy/assisted-installer-controller-ocp/assisted-controller-deployment.yaml

```diff
@@ -17,8 +17,7 @@ spec:
         - name: assisted-installer-controller
           image: REPLACE_CONTROLLER_OCP_IMAGE
           imagePullPolicy: Always
-          command:
-            - /assisted-installer-controller-ocp
+          command: ["assisted-installer-controller-ocp"]
```

Comment:
Shouldn't an absolute path be safer?

Reply:
It isn't obvious to me what using the full path would protect us from. Are we afraid the binary won't be in `PATH`, that we will conflict with some other `assisted-installer-controller-ocp` binary, or something else entirely?

Reply:
The `PATH` issue. But not from a conflict perspective - I guess the binary won't get conflicted with this original name.
But avoiding the working directory or `PATH` configuration dependency.